search-service/src/main/kotlin/com/egm/stellio/search/service/TemporalEntityAttributeService.kt

```diff
@@ -593,12 +593,12 @@ class TemporalEntityAttributeService(
             val currentTea =
                 getForEntityAndAttribute(entityUri, ngsiLdAttribute.name, ngsiLdAttributeInstance.datasetId)
                     .fold({ null }, { it })
+            val attributeMetadata = ngsiLdAttributeInstance.toTemporalAttributeMetadata().bind()
+            val attributePayload = expandedAttributes.getAttributeFromExpandedAttributes(
+                ngsiLdAttribute.name,
+                ngsiLdAttributeInstance.datasetId
+            )!!
             if (currentTea != null) {
```

Comment:
why did you place them outside the if clause ?

Reply:
because I need them in both clauses